extensions/tokenizers/src/main/java/ai/djl/huggingface/tokenizers/jni/LibUtils.java

```diff
@@ -167,7 +167,7 @@ private static boolean copyJniLibraryFromClasspath(
             Utils.moveQuietly(tmp, dir);
             return true;
         } catch (IOException e) {
-            logger.error("Cannot copy jni files", e);
+            logger.debug("Cannot copy jni files: {}", e.getMessage());
```

Comment:
1. We should check null for cu122 case at line 162, it will throw and NPE.
2. We should keep it as warning at least. IOException should not happen.

Reply:
IOException will happen if it's cu122, it will need to download.